MonoGame.Framework/Content/ContentReaders/ModelReader.cs

```diff
@@ -149,9 +149,6 @@ protected internal override Model Read(ContentReader reader, Model existingInsta
                 var parentBoneIndex = ReadBoneReference(reader, boneCount);
 				var boundingSphere = reader.ReadBoundingSphere();
 
-                // Tag
-                reader.ReadObject<object>();
-

```

Comment:
But there is no mesh from ModelMesh class instance before that which is what the code has to write ain't it?